src/conn/conn_oligarch.c

```diff
@@ -49,26 +48,21 @@ __disagg_pick_up_checkpoint(WT_SESSION_IMPL *session, uint64_t checkpoint_id)
         WT_ERR_MSG(session, EINVAL, "Global checkpoint ID went backwards: %" PRIu64 " -> %" PRIu64,
           global_checkpoint_id - 1, checkpoint_id);
 
-    /* Read the checkpoint metadata from the special metadata page. */
+    /*
+     * Part 1: Get the metadata of the shared metadata table and insert it into our metadata table.
+     */
+
+    /* Read the checkpoint metadata of the shared metadata table from the special metadata page. */
     WT_ERR(__wt_disagg_get_meta(session, WT_DISAGG_METADATA_MAIN_PAGE_ID, checkpoint_id, item));
 
-    if (item->size >= sizeof(buf))
```

Comment:
Why we remove this code and the following code?

Reply:
There isn't a need for this code anymore. This code used to say which key in the metadata table to update, but now that we always update `file:WiredTigerShared.wt` with this value, there isn't a need to pass that string across.